perfkitbenchmarker/vm_util.py

```diff
@@ -246,10 +246,10 @@ def WrappedFunction(*args, **kwargs):
           sleep_time = poll_interval * fuzz_multiplier
           if ((time.time() + sleep_time) >= deadline or
               (max_retries >= 0 and tries > max_retries)):
-            raise e
+            raise
```

Comment:
I am not a python expert, but from reading in https://docs.python.org/2/library/logging.html#logger-objects, it may be that the `e` is the same but the `exc_info` that is logged by the higher catch has been updated to the second raise site?